jcm/physics/speedy/shortwave_radiation_test.py

```diff
@@ -159,7 +159,7 @@ def test_shortwave_radiation(self):
         psa = jnp.ones(xy)
         precnv = -1.0 * np.ones(xy)
         precls = 4.0 * np.ones(xy)
-        iptop = 8 * np.ones(xy, dtype=int)
+        iptop = np.ones(xy, dtype=int) * jnp.linspace(0,kx,il).astype(int)[jnp.newaxis,:]
```

Comment:
This is one column with multiple iptops, or multiple columns with changing iptops?